cv32/sim/core/Makefile

```diff
@@ -92,7 +92,7 @@ endif
 
 # xrun is the Cadence xcelium SystemVerilog simulator (https://cadence.com/)
 XRUN              = xrun
-XRUN_FLAGS        = -clean -smartorder -sv -top worklib.tb_top -timescale 1ns/1ps
+XRUN_FLAGS        = -clean -smartorder -sv -disable_sem2009 -top worklib.tb_top -timescale 1ns/1ps
```

Comment:
The structure of the Makefiles has been changed such that simulator dependent variables/targets/rules have their own file.  So the variable XRUN_FLAGS is no longer in the Makefile, it is in `xrun.mk`, which is included by the Makefile.

I recommend issuing a pull-request against xrun,mk if you want to get this switch added.

Reply:
Okay, you can leave this fix out (I did the same thing in two makefiles) and I will make a separate pull request for that. Reason I did above change is that I want to end up in a situation with 0 warnings.